lib/agent/actions/wipe/wipe.js

```diff
@@ -48,7 +48,25 @@ exports.documents = function(cb) {
 }
 
 exports.emails = function(cb) {
-  wipe('emails', cb)
+  wipe('emails', function(err) {
+    if (os_name != 'windows')
+      return cb(err);
```

Comment:
the error for the callback should define something related to the context, such as `cb(Error("can't wipe in " + os_name));`